### PR TITLE
Implement to accept zero as a default value for kubectl create token duration

### LINKED ISSUE #1562: Documentation update. duration flag in kubectl create token
**What would you like to be added**:

I will like to see in the documentation with:

`kubectl create token -h`

not a duration with a value of 0 seconds.

**Why is this needed**:

If we run the command for help it will be shown following text:

```
 --duration=0s:
        Requested lifetime of the issued token. If not set, the lifetime will be determined by the server automatically. The server may return a token with a longer or shorter lifetime.
```

But if i run the command with 0 seconds i will get following error:

`error: --duration must be positive`

### PR DESCRIPTION
**Pull request by ah8ad3**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Talked in bug scrub, Accept zero as a value and act as a default value.
#### Which issue(s) this PR fixes:

Fixes https://github.com/kubernetes/kubectl/issues/1562

#### Does this PR introduce a user-facing change?

```release-note
Accept zero as a default value for kubectl create token duration
```

### CODE REVIEW

File: staging/src/k8s.io/kubectl/pkg/cmd/create/create_token.go [hunk 1]
```diff
@@ -208,7 +208,7 @@ func (o *TokenOptions) Validate() error {
 	if len(o.Namespace) == 0 {
 		return fmt.Errorf("--namespace is required")
 	}
-	if o.Duration < 0 || (o.Duration == 0 && o.Flags.Changed("duration")) {
+	if o.Duration < 0 {
 		return fmt.Errorf("--duration must be positive")
```
Comment (mpuckett159) [inline staging/src/k8s.io/kubectl/pkg/cmd/create/create_token.go:212]: Change this to `--duration must be greater than or equal to 0` for accuracy

Comment (ah8ad3) [reply staging/src/k8s.io/kubectl/pkg/cmd/create/create_token.go]: Good point. Updated, please take a look.

**Comment by ah8ad3** [comment]
Squashed.

**Comment by ardaguclu** [comment]
Thanks!
/approve
I'm deferring to @mpuckett159 for the final tag.

**Comment by mpuckett159** [comment]
Thanks for taking this on @ah8ad3 
/lgtm
/approved

**Pull request merged**